CHANGELOG.md

```diff
@@ -4,6 +4,47 @@
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.27.0...master)
 
+## Megazords
+
+### Breaking Changes
+
+- Megazord initialization is now separate from the initialization of the http client. ([#1066](https://github.com/mozilla/application-services/issues/1066))
+    - This is done to allow application that wish to defer initializing JNA
+      (which can be costly) until later. Now, you should use the `RustHttpConfig.setClient()` function from `org.mozilla.appservices:httpconfig`:
+
+        In `build.gradle`:
+        ```groovy
+        dependencies {
+            // ...
+            implementation "org.mozilla.appservices:httpconfig:${app_services_version}"
+            // ...
+        }
+        ```
+
+        In Kotlin:
+        ```kotlin
+        import mozilla.appservices.httpconfig.RustHttpConfig
+        // At some point after megazord init.
+        RustHttpConfig.setClient(lazy { /* your http client here */ })
```

Comment:
silly question, what happens if the consumers forget to do this?

Reply:
I do not think that this is a silly question -- I think that this is exactly what we are going to have to guard against when using this in Fenix.

Reply:
All network requests fail with a network error, but nothing worse than that. Stuff should start working once you get around to calling RustHttpConfig.setClient.